src/StringTools/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETFramework,Version=v3.5</Target>
```

Comment:
We didn't really drop this TF, did we? Is this a validation error?

Reply:
We did drop the net35 asset from the package just recently: https://github.com/dotnet/msbuild/commit/ee6f71d15eea77c970815d9105ab9520876c5042